e2e/pnpm_lockfiles/lockfile-test.bzl

```diff
@@ -198,8 +201,8 @@ def lockfile_test(npm_link_all_packages, name = None):
             ":.aspect_rules_js/node_modules/lodash@4.17.21/dir",
 
             # Patched dependencies
-            ":.aspect_rules_js/node_modules/meaning-of-life@1.0.0_o3deharooos255qt5xdujc3cuq",
-            "@%s__meaning-of-life__1.0.0_o3deharooos255qt5xdujc3cuq//:pkg" % lock_repo,
```

Comment:
pnpm v10 changed hashes from 26 characters to 64 characters, this causes it to go above the [arbitrary 32 limit we have in rules_js](https://github.com/aspect-build/rules_js/blob/f81d70e46eb83d8c8937cbc2ae20151d0317c065/npm/private/pnpm.bzl#L273-L277) so this becomes a hash (of a hash in this case).

Reply:
why not just increase that arbitrary limit to 64? It feels wrong to me for us to throw away bits of entropy here, since it raises the question "can we have collisions"

Reply:
Technically that would be a "breaking change" if we simply change 32 to 64 because it will change repository names of packages. That is the type of thing I want to explore more while looking into rules_js v3 and I have similar changes in https://github.com/aspect-build/rules_js/pull/2177 atm